doc/UserGuide/CustomizingTheUPP/EnablingOutput.rst

```diff
@@ -0,0 +1,61 @@
+.. _enabling-Output:
+
+********************************
+User-Defined Sigma Level Output
+********************************
+
+This documentation describes the steps to generate model output at user-defined sigma levels using the Unified Post Processor (UPP). The instructions address the configuration of XML files and the resolution of potential issues encountered during this process.
+
+Generating Sigma Level Output
+-----------------------------
+
+UPP can output temperature, U, and V components on sigma surfaces. These correspond to indices 206, 208, and 209 in the :term:`GRIB2`` table. Follow these steps to configure your workflow for this output:
```

Comment:
```suggestion
UPP can output temperature, U, and V components on sigma surfaces. These correspond to indices 206, 208, and 209 in the :term:`GRIB2` table. Follow these steps to configure your workflow for this output:
```

Reply:
Could you actually link to the table? That's more important in terms of knowing what variables are possible in UPP.